ngraph/test/frontend/shared/include/npy.hpp

```diff
@@ -0,0 +1,497 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Comment about this is still valid - it is not acceptable to copy files from other Github repositories and change copyrights to Intel

Reply:
Got it, we will handle this.

Reply:
Actually, this file is copied from ./inference-engine/tests_deprecated/functional/shared_tests/lstm/ , we fill follow their pattern to handle the same third-party dependency